tests/unit/test_paragraphHelper.py

```diff
@@ -0,0 +1,205 @@
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) 2022 NV Access Limited, Rob Meredith
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+
+import unittest
+import documentNavigation.paragraphHelper as paragraphHelper
+import textInfos
+from .textProvider import BasicTextProvider
+
+
+class TestParagraphHelper(unittest.TestCase):
+	def test_singleLineBreakParagraphForward(self):
```

Comment:
I suggest splitting up the naming into underscored groups

```suggestion
	def test_singleLineBreakParagraph_forward(self):
```